### DISCUSSION TITLE
Lattice Transport Bug (norns 2.7.3)

### DISCUSSION THREAD

**Discussion by BeatRossmy** [monome/norns#1640] [General]
I repost a bug report regarding the lattice transport value
@ryleelyman 

What i observed was that the transport received in the actions of individual sprockets did not match with the defined ```ppqn``` and were offset by 2.

When reading through the source code ([norns/lattice.lua at main · monome/norns · GitHub 1](https://github.com/monome/norns/blob/main/lua/lib/lattice.lua)). I discovered that there is probably a bug in function ```Lattice:pulse()```. The incrementation of transport is done within the loop which handles the ordering of sprockets, which leads to a transport value dependent on the defined sprocket order and is multiplied by 5.

The issue is in line 124.

https://github.com/monome/norns/blob/e8ae36069937df037e1893101e73bbdba2d8a3db/lua/lib/lattice.lua#L93-L127

**Comment by robbielyman** [comment]
I believe a fix for this was merged in #1638.
